.github/workflows/evaluate.yml

```diff
@@ -37,11 +37,11 @@ on:
         required: true
         description: 'Dependency packages, you can also set a specific version'
         type: string
-        default: 'pynvml packaging protobuf transformers_stream_generator transformers'
+        default: 'pynvml packaging protobuf transformers_stream_generator transformers human_eval'
```

Comment:
Is `human_eval` a pypi package?

Reply:
> human_eval

yes, It used in humaneval dataset's evaluation.

Reply:
I tried `pip install human_eval`. But there is no such pypi package